src/content/homepage.md

```diff
@@ -2,9 +2,9 @@
 headerSection:
   dark:
     title: Blockchain v3.0
-    subtitle: Alephium is the first operational sharded blockchain bringing scalability,
-      ETH-inspired smart contracts and dApps capabilities to Bitcoin's proven core
-      technologies while ensuring better performance and improved energy efficiency.
+    subtitle: Alephium is the first live layer 1 sharded blockchain scaling 
+    and improving on Bitcoin core technologies, Proof of Work & UTXO. It delivers 
```

Comment:
- I would replace _"...Proof of Work & UTXO"_ with _"...Proof of Work, and UTXO"_.

Reply:
fine by me except the "," after pow